README.md

```diff
@@ -1,14 +1,17 @@
 # SDA-sync
 
-The sda-sync is an integration created to solve the sync/backup issue in BigPicture. In this project, the files (and datasets) that are uploaded in one node, should be synced/backed up in another and the ingestion process should be run on both sides. However, the accession and dataset IDs should be the same in both nodes. Therefore, the integration is taking advantage of the sda-pipeline sync-api in order to send the required data and create the messages needed on the receiving side of the syncing.
+The sda-sync is an integration created to solve the *data* syncing requirement in BigPicture. In this project, every data submission that takes place in a country's node must be *mirrored* in the other country's node. This means that both the submitted data files and their corresponding database identifiers (e.g. accession and dataset ID's) in one country's node should be replicated in the other country's node. This is achieved by syncing the data between the two nodes.
```

Comment:
```suggestion
The sda-sync is an integration created to solve the *data* syncing requirement in BigPicture. In this project, every data submission that takes place in a country's node must be *mirrored* in the other country's node. This means that both the submitted data files and their corresponding database identifiers (e.g. accession and dataset ID's) in one country's node should be replicated in the other country's node. This is achieved by syncing the data between the two nodes and then providing the relevant identifiers for use while ingesting.
```

? I'd prefer having it clearer in this executive summary that the dataset identifiers aren't simple "synced" over.